privacyidea/lib/auditmodules/sqlaudit.py

```diff
@@ -188,9 +216,9 @@ def _create_filter(param, timelimit=None):
                     else:
                         # All other keys are compared as strings
                         column = getattr(LogEntry, search_key)
-                        if search_key == "date":
+                        if search_key.endswith("date"):
                             # but we cast "date" to a string first (required on postgresql)
```

Comment:
Then the comment might be a bit different, now.

Reply:
Done in fcff8608